src/test/java/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -327,11 +327,10 @@ public void filterForStudyAccountNullReturnsNull() {
         assertNull(BridgeUtils.filterForStudy((Account)null));
     }
     
-    @Test
-    public void filterForStudyAccountNoContextNoStudyDoesNotReturnAccount() {
-        RequestContext.set(new RequestContext.Builder().withCallerUserId(TEST_USER_ID).build());
-        assertNull(BridgeUtils.filterForStudy(getAccountWithStudy()));
-    }
```

Comment:
Look at this again, you put a context here despite it saying this is a no context test.

Reply:
Added back a test that *really* has a null context, and verified that it returns the account in that case.